scheduler/src/cook/mesos/api.clj

```diff
@@ -2705,8 +2705,8 @@
                                  (not (string? body)))
                             (coll? body))]
       (cond-> resp
-        json-response (res/content-type "application/json")
-        (and json-response body) (assoc :body (streaming-json-encoder body))))))
+              json-response (res/content-type "application/json")
```

Comment:
`cond->` conditions and clauses should be indented one level. You can change Cursive settings to fix this.

Reply:
Reverted